### PR TITLE
Reword long sentence

### LINKED ISSUE #723: Long sentence in definition of public code
This paragraph long sentence from[ the glossary definition of public code](https://about.publiccode.net/glossary/public-code-definition.html) is a bit hard to parse:

> By developing public code independent from but implementable in a local context, as well as documenting the development process openly, it can provide a building block for others to re-implement in their local context, take as a starting point to continue development or as a basis for learning.

I suggest we make it easier to read by splitting up into several sentences. Suggestion:

> Public code can provide a building block for others to re-implement in their local context. It can serve as a starting point to continue development or as a basis for learning. To enable that, the development of public code needs to be independent from, but implementable in, a local context. The development process must also be documented openly.

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #723

Making incremental fix instead of trying to solve other things mentioned in the issue as well.

-----
[View rendered glossary/public-code-definition.md](https://github.com/publiccodenet/about/blob/ja-publiccode-definition-fix/glossary/public-code-definition.md)

### CODE REVIEW

File: glossary/public-code-definition.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ Public code is both civic code (like policy or regulation) and computer source c
 
 Because public code serves the public interest, it should be: open, legible, accountable, accessible and sustainable.
 
-By developing public code independent from but implementable in a local context, as well as documenting the development process openly, it can provide a building block for others to re-implement in their local context, take as a starting point to continue development or as a basis for learning.
+Public code can be building blocks for others to re-use and re-implement in their local context, creating their public code. It can serve as a starting point to continue development or as a basis for learning. To enable that, the development of public code needs to be independent from, but implementable in, those contexts. The development process must also be openly documented.
```
Comment (lscheske) [inline glossary/public-code-definition.md:11]:
```suggestion
Public code can comprise building blocks for others to re-use and implement in their local context, creating new instances of public code. It can serve as a starting point to continue development or as a basis for learning. To enable that, public code development needs to be independent from, but implementable in different contexts. The development process must also be openly documented.
```

**Review by lscheske — CHANGES_REQUESTED** [review]
I polished the language slightly, hope this is clear.

File: glossary/public-code-definition.md [hunk 2]
```diff
@@ -8,6 +8,6 @@ Public code is both civic code (like policy or regulation) and computer source c
 
 Because public code serves the public interest, it should be: open, legible, accountable, accessible and sustainable.
```
Comment (ElenaFdR) [inline glossary/public-code-definition.md:9]:
```suggestion
Because public code serves the public interest, it should be open, legible, accountable, accessible and sustainable.
```

File: glossary/public-code-definition.md (hunk 1 above)
Comment (ElenaFdR) [inline glossary/public-code-definition.md:11]:
```suggestion
Public code can provide a building block for others to:

* use as a basis for learning
* take as a starting point to continue development
* re-implement in their local context

To enable this, public code development needs to be independent from, but implementable in all of the contexts listed above. The development process must also be openly documented.
```

**Comment by ElenaFdR** [comment]
@Ainali @lscheske I've made an alternate suggestion for the long sentence!

File: glossary/public-code-definition.md (hunk 1 above)
Comment (Ainali) [reply glossary/public-code-definition.md:11]: This sounds confusing to me, as it seems like the coe has agency providing something else rather than being the building block itself (which I still believe is the point we want to make, right?).

Comment (ElenaFdR) [reply glossary/public-code-definition.md:11]: I've updated this - is it clearer now? I'm trying to avoid @lscheske's 'comprise' suggestion because (for me) that suggests that the building blocks are part of a larger set that we're not explaining.

**Review by lscheske — APPROVED** [review]
